osquery/remote/enroll/plugins/tls.cpp

```diff
@@ -20,6 +22,12 @@
 #include "osquery/core/process.h"
 
 namespace osquery {
+namespace tables {
+QueryData genOsqueryInfo(QueryContext& context);
```

Comment:
If you see this style of "I need to use a table" anywhere in the code base please fix it, it's poor design but we do use it in a few cases.

We should avoid "bind" tables to "core" osquery components. Instead prefer to use the `SQL` class.